src/Explorer/Controls/Settings/SettingsSubComponents/MongoIndexingPolicy/MongoIndexingPolicyComponent.tsx

```diff
@@ -282,6 +283,15 @@ export class MongoIndexingPolicyComponent extends React.Component<MongoIndexingP
     );
   };
 
+  public hasCompoundIndex = (): boolean => {
+    for (let index = 0; index < this.props.mongoIndexes.length; index++) {
+      if (this.props.mongoIndexes[index].key.keys.length > 1) {
```

Comment:
>keys [](start = 45, length = 4)

can this be null?

Reply:
according to the data model, it can. though I haven't seen any nulls being sent for this from RP. will add the null check